src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -1,279 +1,400 @@
+import AggregationTable from "./AggregationTable";
+import { v4 as uuidv4 } from "uuid";
+import { UserSchema } from "../../schemas/UserSchema";
 
-//TODO - Refactor to backend calls once setup to pull rows, etc. 
+import { SearchIcon, WarningIcon, DownloadIcon } from "@chakra-ui/icons";
+import { Select, components } from "chakra-react-select";
 
+//TODO - Refactor to backend calls once setup to pull rows, etc.
 
-//To test uploading a timesheet 
-// apiClient.updateUserTimesheet(testingTimesheetResp); 
+//To test uploading a timesheet
+// apiClient.updateUserTimesheet(testingTimesheetResp);
 
 const createEmptyTable = (startDate, company) => {
-    // We assign uuid to provide a unique key identifier to each row for reacts rendering 
-    
-    //TODO's: Pull UserID automatically
-    return {
-        UserID: "77566d69-3b61-452a-afe8-73dcda96f876",
-        TimesheetID: uuidv4(), 
-        CompanyID: company, 
-        StartDate: startDate, 
-        Status: {
-            Stage: Review_Stages.APPROVED, 
-            Timestamp: undefined 
-        },
-        WeekComments: [], 
-        TableData:[], 
-        ScheduledData: undefined 
-    }
-}
-
-const user = 'Example User'
+  // We assign uuid to provide a unique key identifier to each row for reacts rendering
+
+  //TODO's: Pull UserID automatically
+  return {
+    UserID: "77566d69-3b61-452a-afe8-73dcda96f876",
+    TimesheetID: uuidv4(),
+    CompanyID: company,
+    StartDate: startDate,
+    Status: {
+      Stage: Review_Stages.APPROVED,
+      Timestamp: undefined,
+    },
+    WeekComments: [],
+    TableData: [],
+    ScheduledData: undefined,
+  };
+};
+
+const user = "Example User";
 
 const testingEmployees = [
-    {UserID: "abc", FirstName: "joe", LastName: "jane", Type: "Employee", Picture: "https://www.google.com/koala.png"},
-    {UserID: "bcd", FirstName: "david", LastName: "lev", Type: "Employee", Picture: "https://www.google.com/panda.png"},
-    {UserID: "cde", FirstName: "crys", LastName: "tal", Type: "Employee", Picture: "https://www.google.com/capybara.png"},
-    {UserID: "def", FirstName: "ken", LastName: "ney", Type: "Employee", Picture: "https://www.google.com/koala.png"},
-]
-
-function ProfileCard({employee}) {
-
+  {
+    UserID: "abc",
+    FirstName: "joe",
+    LastName: "jane",
+    Type: "Employee",
+    Picture: "https://www.google.com/koala.png",
+  },
+  {
+    UserID: "bcd",
+    FirstName: "david",
+    LastName: "lev",
+    Type: "Employee",
+    Picture: "https://www.google.com/panda.png",
+  },
+  {
+    UserID: "cde",
+    FirstName: "crys",
+    LastName: "tal",
+    Type: "Employee",
+    Picture: "https://www.google.com/capybara.png",
+  },
+  {
+    UserID: "def",
+    FirstName: "ken",
+    LastName: "ney",
+    Type: "Employee",
+    Picture: "https://www.google.com/koala.png",
+  },
+];
+
+
+// size - s,m,l
+function AuthorComponent({ user , size }) {
+
+  const fullName = user?.FirstName + " " + user?.LastName
```

Comment:
Is there a case where it makes sense to pass a null/undefined user? Should we just not show the component?

Reply:
The user state only gets set after the the first useEffect runs so when the page initially loads the component takes in an undefined user, I changed it to check if the user is null/undefined and not display if so but I'm not sure if this is the best way to handle that.

Reply:
It's mostly a UX design decision - an alternative would be to show a loading icon, but maybe you'd want one loading icon for the whole page while hiding this component (and if we do a loading icon, we'd need another state for isFetching to distinguish between undefined bc fetching vs error). You could also not leave it to this component and instead ensure that the entire component only renders if user is defined, ex:
```
// in Page
{user ? <AuthorComponent user={user} /> : isFetching ? <LoadingIcon /> : <ErrorDisplay />}
// or
{user && <AuthorComponent user={user} />}
{isFetching ? <PageLoadingIcon />}
```

You might want to reach out to Ella about visualizing loading/errors. You can incorporate that in this PR or leave it as just hiding the component and open a ticket for loading/error handling.